src/features/SignAndVerifyMessage/SignMessage.tsx

```diff
@@ -194,36 +143,52 @@ function SignMessage(props: Props) {
             label={translate('MSG_MESSAGE')}
             placeholder={translateRaw('SIGN_MSG_PLACEHOLDER')}
             textarea={true}
-            onChange={(event) => handleOnChange(event.target.value)}
+            onChange={(event) => messageUpdate(event.target.value)}
             height="150px"
-            inputError={error}
+            inputError={error && translateRaw('SIGN_MESSAGE_ERROR')}
           />
           <SignButton
             disabled={!message || isDemoMode}
             onClick={handleSignMessage}
-            loading={signStatus === SignStatus.SIGNING}
+            loading={status === 'SIGN_REQUEST'}
           >
-            {signStatus === SignStatus.SIGNING ? translate('SUBMITTING') : translate('NAV_SIGNMSG')}
+            {status === 'SIGN_REQUEST' ? translate('SUBMITTING') : translate('NAV_SIGNMSG')}
```

Comment:
Can you use `SignStatus` here?

Reply:
see https://github.com/MyCryptoHQ/MyCrypto/pull/3962#discussion_r639918293